kornia/augmentation/augmentation.py

```diff
@@ -467,6 +447,12 @@ class RandomPerspective(GeometricAugmentationBase2D):
         tensor([[[[0.0500, 0.0961, 0.0000],
                   [0.2011, 0.3144, 0.0000],
                   [0.0031, 0.0130, 0.0053]]]])
+
+    To apply the exact augmenation again, you may take the advantage of the previous parameter state:
+        >>> input = torch.randn(1, 3, 32, 32)
+        >>> aug = RandomPerspective(0.5, p=1.)
+        >>> (aug(input) == aug(input, params=aug._params)).all()
```

Comment:
How’s it working right now. Are the parameters generated before nowing the input shape?